src/infrastructure/delivery/provider/entities.ts

```diff
@@ -0,0 +1,18 @@
+import { Mail } from './Mail';
+
+export type ProviderResponse = {
```

Comment:
As above, what do you think if we use T like suffix for the types

Reply:
https://github.com/GitbarPodcast/Botletter/blob/feature-mailchimp-integration/src/infrastructure/delivery/provider/entities.ts#L11